hm_1/tasks/task_02_01.py

```diff
@@ -0,0 +1,65 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from collections import Counter
+from typing import List
+
+
+def get_text(file_path: str) -> str:
+    data = ""
+    with open(file_path, encoding="unicode_escape") as f:
+        for line in f:
+            data += line
+    return data
+
+
+def get_longest_diverse_words(file_path: str) -> List[str]:
+    data = {}
+    text_without_punctuation = get_text(file_path).translate(
+        str.maketrans("", "", string.punctuation)
+    )
+    words = text_without_punctuation.split()
+    for i in words:
+        data[i] = len(set(i.lower()))
+    size_of_data = len(data) if len(data) < 10 else 10
```

Comment:
Why do we need this variable?  Can we do better?

Reply:
This variable need for cases with files, which consists less than 10 words. I don’t know how avoid this without it

Reply:
I am pretty sure you missed some pieces of how slices work in Python. Try to experiment with the list slicing in the terminal.